tests/Unit/Helpers/GoodieTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Test\Unit\Helpers;
+
+use Engelsystem\Config\Config;
+use Engelsystem\Helpers\Goodie;
+use Engelsystem\Models\User\User;
+use Engelsystem\Models\Worklog;
+use Engelsystem\Test\Unit\HasDatabase;
+use Engelsystem\Test\Unit\TestCase;
+use Illuminate\Database\Query\Grammars\SQLiteGrammar;
+
+class GoodieTest extends TestCase
```

Comment:
please also write a test with night shifts and freeloads since you already had a issue in your calculations with this

Reply:
At the moment that code path is excluded as the query can't be run on sqlite

Reply:
Its tested now as a feature test and thus not that stable but works ;)